Description: Remove obsolete ssconf files
  Debian packaging relies on obsolete ssconf files to detect whether the
  upgrade has completed or not; this is not needed for upstream,
  although it is a good cleanup
Author: Iustin Pop <iustin@debian.org>
Last-Update: 2010-10-24
--- a/tools/cfgupgrade12
+++ b/tools/cfgupgrade12
@@ -404,6 +404,10 @@
         logging.debug("Writing RAPI certificate to %s", options.RAPI_CERT_FILE)
         utils.GenerateSelfSignedSslCert(options.RAPI_CERT_FILE)
 
+      ss_hyp = SsconfName('hypervisor')
+      if os.path.exists(ss_hyp):
+        utils.RemoveFile(ss_hyp)
+
   except:
     logging.critical("Writing configuration failed. It is probably in an"
                      " inconsistent state and needs manual intervention.")
